src/wxflow/configuration.py

```diff
@@ -107,8 +112,9 @@ def _get_shell_env(scripts: List) -> Dict[str, Any]:
         runme = ''.join([f'source {s} ; ' for s in scripts])
         magic = f'--- ENVIRONMENT BEGIN {random.randint(0,64**5)} ---'
         runme += f'/bin/echo -n "{magic}" ; /usr/bin/env -0'
+        bash_path = shutil.which('bash')
```

Comment:
Is there a reason why this is necessary?

Reply:
Yes, In the Git Hub Runner the subprocess shell was defaulting to `sh` and would error on `source`.  I used **which** here just to make sure when designating bash that we didn't run into` /usr/bin/bash` vs. `/bin/bash`.

Reply:
would replacing `source` with `.` eliminate the need for specifying a shell?  
I ask because what if the configs have `#!/bin/ksh` or `#!/bin/csh`.

Reply:
we will assume for now that our config files are `bash`